WooCommerce/UITestsFoundation/Screens/Orders/PaymentMethodsScreen.swift

```diff
@@ -0,0 +1,40 @@
+import ScreenObject
+import XCTest
+
+public final class PaymentMethodsScreen: ScreenObject {
+
+    private let cardMethodButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["payment-methods-view-card-row"]
+    }
+
+    private let dismissButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["payment-methods-view-cancel-button"]
+    }
+
+    /// Button to dismiss a the modal
+    ///
+    private var dismissButton: XCUIElement { dismissButtonGetter(app) }
+
+    /// Button to dismiss a the modal
```

Comment:
super nit from my understanding of the button:
```suggestion
    /// Button to select card payment method
```